scripts/build_libkrun.sh

```diff
@@ -143,6 +144,16 @@ check_linux_packages() {
                 fi
             done
             ;;
+				alpine)
+						install_command="apk add"
+						packages=$ALPINE_PACKAGES
+						# Check if apk command exists and packages are installed
+						for pkg in $packages; do
+								if ! apk info -e "$pkg" >/dev/null 2>&1; then
+										missing_packages="$missing_packages $pkg"
+								fi
+						done
+						;;
```

Comment:
I have reviewed your other [other PR](https://github.com/microsandbox/libkrun/pull/16). 

The indentation is uneven here. The file uses 4-space indents